.github/workflows/e2etest.yml

```diff
@@ -1,61 +1,61 @@
 name: e2e-tests
 
 on:
-  pull_request:
```

Comment:
there are so many changes due to formatting, can u describe the actual change

Reply:
@kalagarraj Sure, thank you for asking. In the step "Auth Devhub" I have just changed 
```
          echo ${{ env.DEVHUB_AUTH }} > authFile
          sf force auth sfdxurl store -f 'authFile'
```

to 

```
          echo '${{ env.DEVHUB_AUTH }}' > authFile
          sf force auth sfdxurl store -f authFile
```

this fixed the inconsistency happened in some PR's where the authFile was not read correctly.